src/main/kotlin/com/craftschemers/hub/minigame/oitq/OneInTheQuiverGameData.kt

```diff
@@ -0,0 +1,29 @@
+package com.craftschemers.hub.minigame.oitq
+
+import com.craftschemers.hub.HubPlayer
+import com.craftschemers.hub.minigame.AbstractGameData
+import com.craftschemers.hub.minigame.GameState
+import com.craftschemers.hub.minigame.GameType
+import com.craftschemers.hub.tasks.StartGameTask
+
+class OneInTheQuiverGameData : AbstractGameData() {
+
+    private val startGameTask = StartGameTask(this, OneInTheQuiver)
+
+    override fun addPlayer(player: HubPlayer) {
+        if (state == GameState.LOBBY && players.size < OneInTheQuiver.maxPlayers) {
```

Comment:
Again, there should probably be some error case handling here instead of silently failing.